apps/auth0-rules/package.json

```diff
@@ -10,17 +10,17 @@
     "watch:babel": "../../scripts/build-babel.sh watch"
   },
   "dependencies": {
```

Comment:
Gotta remember to push any changes to the built rules to auth0 (manual process) and check if everything works.

Reply:
(done this already)